rero_ils/modules/documents/dojson/contrib/marc21tojson/model.py

```diff
@@ -290,11 +290,9 @@ def marc21_to_publishers_publicationDate(self, key, value):
                 self['freeFormedPublicationDate'] = data
         indexes[tag] = index + 1
         lasttag = tag
-    publishers.append(publisher)
-    if publishers == [{}]:
-        return None
-    else:
-        return publishers
+
+    if publisher:
+        publishers.append(publisher)
```

Comment:
I think your are missing `return something`?

Reply:
Yes of course, sorry..